src/index.css

```
@@ -44,14 +44,14 @@
 @font-face {
-  font-family: "Cafe24GounNight";
-  src: url("/fonts/Cafe24GounNight.woff2") format("woff2");
+  font-family: 'Cafe24GounNight';
+  src: url('/fonts/Cafe24GounNight.woff2') format('woff2');
   font-display: swap;
 }
 
 @font-face {
-  font-family: "LeeSeoYoon";
-  src: url("/fonts/LeeSeoYoon.woff2") format("woff2");
+  font-family: 'LeeSeoYoon';
```

Stylelint (17.9.0)

[error] Expected no quotes around "LeeSeoYoon" (font-family-name-quotes)

(font-family-name-quotes)